src/plugins/data/common/field_formats/converters/duration.ts

```diff
@@ -294,7 +294,7 @@ function formatDuration(
 
   const getUnitText = (method: string) => {
     const type = outputFormats.find(({ method: methodT }) => method === methodT);
-    return useShortSuffix ? type?.shortText : type?.text;
+    return useShortSuffix ? type?.shortText : type?.text.toLowerCase();
```

Comment:
```suggestion
    return useShortSuffix ? type?.shortText : type?.text?.toLowerCase();
```

I'd add `?` check just to be extra safe and make sure we won't catch `type error` here